account_multicompany_easy_creation/__manifest__.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2018 Carlos Dauden - Tecnativa <carlos.dauden@tecnativa.com>
+# Copyright 2022 Moduon - Eduardo de Miguel
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    "name": "Quick Company Creation Wizard",
+    "summary": "This module adds a wizard to create companies easily",
+    "version": "16.0.1.0.0",
+    "category": "Multicompany",
+    "website": "https://github.com/OCA/multi-company"
+    "account_multicompany_easy_creation",
+    "author": "Tecnativa, Moduon, Odoo Community Association (OCA)",
```

Comment:
I don't think we should be considered authors just by migrating the module. Of course, we're contributors, but authorship means designing the module from scratch, and that wasn't our case.

I might be wrong, as there's no clear explanation about this difference in the contributors guide, but I think it's fair to leave the original authorship, at least unless a deep refactor is done.

Could you revert this please?

```suggestion
    "author": "Tecnativa, Odoo Community Association (OCA)",
```